urllib3/util/typing.py

```diff
@@ -0,0 +1,57 @@
+try:
+    from typing import (
```

Comment:
The CI do not complain about it, also, there is not `typing` modules in python2 (mypy works from python3.2).